sonar-scanner-engine/src/test/java/org/sonar/scanner/mediumtest/ScannerMediumTester.java

```diff
@@ -300,6 +303,7 @@ public TaskResult execute() {
           tester.projectRefProvider,
           tester.activeRules,
           tester.serverIssues,
+          mock(ScmConfiguration.class),
```

Comment:
I'm surprised we have to inject a mock for medium tests. The purpose is to test real components as far as possible.